### PR TITLE
Changed all printf()'s to the appropriate LOG_XXX()'s

### PR DESCRIPTION
**Pull request by johnharbold**

### CODE REVIEW

File: meta-facebook/yv35-bb/src/platform/plat_isr.c [hunk 1]
```diff
@@ -275,7 +274,7 @@ void set_sled_cycle()
 	msg.data[1] = 0x01; // value
 
 	if (i2c_master_write(&msg, retry) < 0) {
-		printf("sled cycle fail\n");
+		LOG_ERR("sled cycle fail");
```
Comment (garnermic) [inline meta-facebook/yv35-bb/src/platform/plat_isr.c:277]: Sled

File: meta-facebook/yv35-cl/src/platform/plat_class.c [hunk 2]
```diff
@@ -367,11 +369,14 @@ void init_platform_config()
...
-				printf("Unknown the 2OU card type, the card type read from CPLD is 0x%x\n",
+				LOG_ERR("Unknown the 2OU card type, the card type read from CPLD is 0x%x",
 				       i2c_msg.data[0]);
 				break;
 			}
 		}
 	}
 	SAFE_FREE(data);
 }
+/*
```
Comment (garnermic) [inline meta-facebook/yv35-cl/src/platform/plat_class.c:380]: Not sure why this is here.

File: meta-facebook/yv35-cl/src/platform/plat_pmic.c [hunk 3]
```diff
@@ -72,7 +72,7 @@ static uint8_t pmic_i3c_addr_list[MAX_COUNT_DIMM / 2] = { PMIC_A0_A4_ADDR, PMIC_
 
 void start_monitor_pmic_error_thread()
 {
-	LOG_INF("Start thread to monitor PMIC error");
+	LOG_ERR("Start thread to monitor PMIC error");
```
Comment (garnermic) [inline meta-facebook/yv35-cl/src/platform/plat_pmic.c:75]: LOG_INF

File: meta-facebook/yv35-cl/src/platform/plat_sdr_table.c [hunk 4]
```diff
@@ -24,6 +24,14 @@
 #include "plat_class.h"
 #include "plat_ipmb.h"
 #include "plat_sensor_table.h"
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(plat_sdr_table);
+
+
+#include <logging/log.h>
```
Comment (garnermic) [inline meta-facebook/yv35-cl/src/platform/plat_sdr_table.c:32]: Remove duplicate header and LOG_MODULE_REGISTER

File: meta-facebook/yv35-cl/src/platform/plat_sensor_table.c [hunk 5]
```diff
@@ -618,7 +620,7 @@ bool pal_is_time_to_poll(uint8_t sensor_num, int poll_time)
 		}
 	}
 
-	printf("[%s] can't find sensor 0x%x last accest time\n", __func__, sensor_num);
+	LOG_ERR("can't find sensor 0x%x last accest time", sensor_num);
```
Comment (garnermic) [inline meta-facebook/yv35-cl/src/platform/plat_sensor_table.c:623]: Can't

File: meta-facebook/yv35-rf/src/ipmi/plat_ipmi.c [hunk 6]
```diff
@@ -25,7 +25,7 @@
 void OEM_1S_GET_BOARD_ID(ipmi_msg *msg)
 {
 	if (msg == NULL) {
-		printf("%s failed due to parameter *msg is NULL\n", __func__);
+		LOG_ERR(" failed due to parameter *msg is NULL");
```
Comment (garnermic) [inline meta-facebook/yv35-rf/src/ipmi/plat_ipmi.c:28]: Failed

File: meta-facebook/yv35-rf/src/platform/plat_hook.c [hunk 7]
```diff
@@ -116,7 +116,7 @@ bool pre_vr_read(uint8_t sensor_num, void *args)
 
 	ret = i2c_master_write(&msg, retry);
 	if (ret != 0) {
-		printf("[%s] i2c write fail  ret: %d\n", __func__, ret);
+		LOG_ERR("i2c write fail  ret: %d", ret);
```
Comment (garnermic) [inline meta-facebook/yv35-rf/src/platform/plat_hook.c:119]: I2C

File: meta-facebook/yv35-rf/src/platform/plat_hook.c [hunk 8]
```diff
@@ -145,7 +145,7 @@ bool pre_isl69254iraz_t_read(uint8_t sensor_num, void *args)
...
```
Comment (garnermic) [inline meta-facebook/yv35-rf/src/platform/plat_hook.c:148]: I2C

**Comment by johnharbold** [comment]
Hi,

Sorry, but I'm not a Meta employee.

John
________________________________

**Comment by johnharbold** [comment]
Mike,

Thanks for this email. These is the requirements I really need.  Peter F. gave some pretty vague requirements before he left Meta.

I'm going to look at some questionable  uses of memcpy().  Some of the memcpy()'s I saw, I put some asserts to check that the buffers are not being overrun.  Bad software mojo!

John
________________________________

**Pull request closed without merging**